src/Coordination/KeeperServer.cpp

```diff
@@ -161,6 +161,24 @@ struct KeeperServer::KeeperRaftServer : public nuraft::raft_server
         reconfigure(new_config);
     }
 
+    uint64_t getFollowersCount()
+    {
+        if (!is_leader())
+        {
+            return 0;
+        }
```

Comment:
we assume we are leader if we are asking for this information so this could lead to weird results for `mntr` command.

Reply:
For `mntr` command, will return the `follower_count/synced_follower_count` only when this server is leader.
So there will not be misunderstanding here.
BTW, in the original implementation using `get_peer_info_all().size()`, if the server is not leader, 0 will be returned either.

Reply:
the problem is that `is_leader` is an atomic. When we call `mntr` command, we check the atomic for the first time, we get true, and than we check it the second time and get false.
We would get incorrect result (`is_leader = 1`, `follower_count = 0`). 
You are correct that `get_peer_info_all()` will return empty in that case also, so maybe we can iterate manually `peers_` under the lock and return the count regardless if we are currently leader.

Reply:
@lingpeng0314 any updates here?

Reply:
Closing.